Library/Formula/liblaxjson.rb

```diff
@@ -0,0 +1,27 @@
+
+class Liblaxjson < Formula
+  desc "C library for parsing relaxed JSON config file"
+  homepage "https://github.com/andrewrk/liblaxjson"
+  url "https://github.com/andrewrk/liblaxjson/archive/1.0.5.tar.gz"
+  version "1.0.5"
+  sha256 "ffc495b5837e703b13af3f5a5790365dc3a6794f12f0fa93fb8593b162b0b762"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "mkdir", "build"
+    cd "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make", "install"
+    end
+  end
+
+  test do
+    system "mkdir", "build"
+    cd "build" do
+      system "echo", "`pwd`"
+      system "cmake", *testpath, *std_cmake_args
```

Comment:
You can’t rely on `cmake` being available outside of the `install` method. See [`tinyxml2.rb`](https://github.com/Homebrew/homebrew/blob/599bc69a0e2e20a66d0234b56ac14a65f2e9f7a0/Library/Formula/tinyxml2.rb#L23-L33) for an example of a library formula with a good test.